ztools/ztls/handshake_client.go

```diff
@@ -40,9 +42,11 @@ type ClientFingerprintConfiguration struct {
 	// Version in the handshake header
 	HandshakeVersion uint16
 
-	// if len == 32, it will specify the client random
+	// if len == 32, it will specify the client random.
 	// Otherwise, the field will be random
-	ClientRandom []byte
+	// except the top 4 bytes if InsertTimestamp is true
+	ClientRandom    []byte
+	InsertTimestamp bool
```

Comment:
This seems like a good thing to do, as far as camouflaging clients go. Android does this, apparently.